web/src/features/fireWeather/slices/parseModelValuesHelper.ts

```diff
@@ -0,0 +1,34 @@
+import { ModelRun, ModelValue } from 'api/modelAPI'
+import { isNoonInPST } from 'utils/date'
+
+export const parseModelValuesHelper = (
+  model_runs: ModelRun[],
+  separate_noon_values: boolean
+): Record<string, ModelValue[]> => {
+  const pastModelValues: ModelValue[] = []
+  const modelValues: ModelValue[] = []
+  const noonModelValues: ModelValue[] = []
+  const allModelValues = model_runs.reduce(
+    (values: ModelValue[], modelRun) => values.concat(modelRun.values),
+    []
+  )
+  const currDate = new Date()
+  allModelValues.forEach(v => {
+    const isFutureModel = new Date(v.datetime) >= currDate
+    if (isFutureModel) {
+      modelValues.push(v)
+    } else {
+      pastModelValues.push(v)
+      if (separate_noon_values && isNoonInPST(v.datetime)) {
+        noonModelValues.push(v)
+      }
```

Comment:
Just realized this block has to be a separate if statement because we want to include future model values in `noonModelValues` as well

Reply:
gotcha. Fixed in my latest commit